### PR TITLE
optimises load time

### PR DESCRIPTION
**Pull request by Bod9001**
Removed some find object of type, and split out a Initial Functions of a few frames

### CODE REVIEW

File: UnityProject/Assets/Scripts/Managers/CentComm.cs [hunk 1]
```diff
@@ -78,11 +79,14 @@ public void UpdateStatusDisplay(StatusDisplayChannel channel, string text)
 	private string AntagInitialUpdate =
 		"Enemy communication intercepted. Security level elevated.";
 
-	void Start()
+	public InitialisationSystems Subsystem => InitialisationSystems.CentComm;
+
+	void IInitialise.Initialise()
 	{
 		paperPrefab = Resources.Load<GameObject>("Paper");
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Managers/CentComm.cs:86]: this should probably be a GameObject field rather than magical string loading.

Comment (corp-0) [reply UnityProject/Assets/Scripts/Managers/CentComm.cs:86]: I know, not part of this PR. Just commenting for later.

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Managers/CentComm.cs:86]: good catch

**Comment by Bod9001** [comment]
build didn't work so well

**Pull request merged**